tests/test_ntp_config.rb

```diff
@@ -26,28 +26,75 @@ def setup
     no_ntpconfig
   end
 
+  def teardown
+    # teardown runs at the end of each test
+    no_ntpconfig
+    super
+  end
+
   def no_ntpconfig
     # Turn the feature off for a clean test.
     if platform == :ios_xr
-      config("no ntp source #{interfaces[0]}")
+      config("no ntp source #{interfaces[1]}")
```

Comment:
The test was using `interface[1]` but the teardown was not.  This was leaving cruft behind after the tests were complete.